### PR TITLE
Compatible with Symfony3

### PR DESCRIPTION
**Pull request by Big-Shark**
https://github.com/propelorm/Propel2/pull/1060

### CODE REVIEW

**Comment by havvg** [comment]
I restarted the Travis Build as Propel2#1060 has been merged, but it's failing with Symfony 3.0: https://travis-ci.org/propelorm/PropelBundle/jobs/96093241

**Comment by Gamesh** [comment]
in sf 3 it appears that ACL component is not includes by default.
`composer require  "symfony/security-acl": "3.0.x-dev"` resolve this

**Comment by Big-Shark** [comment]
Hmmm, class ObjectChoiceList removed in 3.0, I think we should skip test, and wrote comment about this situation, because I don't know how I need rewrite ModelChoiceList.
Or it may we should create branch 3?
What do you think about it?

**Comment by Maxooo** [comment]
+1 @Big-Shark I think it's necessary to create a new branch for the third version as Symfony 3 changed important things (like the options in the ChoiceList)

**Comment by angelk** [comment]
+1 for the new branch for symfony3

**Comment by havvg** [comment]
What do you think of dropping support for Propel2 for Symfony versions <2.8? Ping @marcj

I think we should focus on supporting Propel2 with 2.8 LTS and 3.0, so we can skip the older versions right away. I'm suggesting this, because Propel2 is still alpha :)

**Comment by Big-Shark** [comment]
@havvg +1 Good idea.

**Comment by gharlan** [comment]
> I think we should focus on supporting Propel2 with 2.8 LTS and 3.0

:+1:

**Comment by havvg** [comment]
@Big-Shark could you rebase this onto 2.0?

**Comment by Big-Shark** [comment]
@havvg Damn, I accidentally erased my branch (

**Comment by havvg** [comment]
`git reflog` and you can get it back :)

**Comment by havvg** [comment]
Thanks :)

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
     "require": {
         "propel/propel": "dev-master",
-        "symfony/symfony": "^2.8"
+        "symfony/symfony": "~2.5||~3.0"
```
Comment (gharlan) [inline composer.json:19]: PropelBundle 2.0 is not compatible to symfony < 2.8

File: .travis.yml [hunk 2]
```diff
@@ -24,8 +24,13 @@ matrix:
...
         - php: hhvm
-        - env: SYMFONY_VERSION="^3.0.0"
-        - env: SYMFONY_VERSION="^3.0.0" COMPOSER_FLAGS="--prefer-lowest"
+        - env: SYMFONY_VERSION="dev-master"
+    exclude:
+        # Symfony 3.x requires PHP 5.5
+        - php: 5.4
+          env: SYMFONY_VERSION="~3.0.0"
+        - php: 5.4
+          env: SYMFONY_VERSION="dev-master"
```
Comment (gharlan) [inline .travis.yml:33]: unneeded exclude part, we do not test php 5.4, because propel 2 requires php 5.5

File: .travis.yml [hunk 3]
```diff
@@ -24,8 +24,13 @@ matrix:
 
     allow_failures:
         - php: hhvm
-        - env: SYMFONY_VERSION="^3.0.0"
-        - env: SYMFONY_VERSION="^3.0.0" COMPOSER_FLAGS="--prefer-lowest"
+        - env: SYMFONY_VERSION="dev-master"
```
Comment (gharlan) [inline .travis.yml:27]: `SYMFONY_VERSION="dev-master"` is not in build matrix, so it does not make sense to allow failures for that.

**Pull request merged**